src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -204,7 +224,8 @@ public enum MenuBarType
         Edit,
         View,
         Help,
-        Packages
+        Packages,
+        Extensions
```

Comment:
Can you add a TODO to the cases above mentioning these will be deleted in Dynamo 3.0?